mailbox/api/src/main/java/org/apache/james/mailbox/events/MailboxListener.java

```diff
@@ -587,22 +588,32 @@ public final int hashCode() {
      */
     class FlagsUpdated extends MessageEvent {
         private final List<MessageUid> uids;
+        private final List<MessageId> messageIds;
```

Comment:
Why do we need this additional fields if computing messageIds from updated flags is easy?

Reply:
IMO BTW idem for uids

Reply:
Will check that. Regarding uids that will depends of the effort it needs, it was already there after all

Reply:
Sounds right actually, thanks